x/wasm/keeper/bench_test.go

```diff
@@ -56,3 +56,34 @@ func BenchmarkExecution(b *testing.B) {
 		})
 	}
 }
+
+func BenchmarkAPI(b *testing.B) {
+	wasmConfig := types.WasmConfig{MemoryCacheSize: 0}
+	ctx, keepers := createTestInput(b, false, SupportedFeatures, nil, nil, wasmConfig, memdb.NewDB())
+	example := InstantiateHackatomExampleContract(b, ctx, keepers)
+	api := keepers.WasmKeeper.cosmwasmAPI(ctx)
+	addrStr := example.Contract.String()
+	addrBytes, err := sdk.AccAddressToBytes(example.Contract.String())
+	require.NoError(b, err)
+	b.Run("GetContractEnv", func(b *testing.B) {
+		b.ResetTimer()
+		for i := 0; i < b.N; i++ {
+			_, _, _, _, _, _, _, err := api.GetContractEnv(addrStr)
+			require.NoError(b, err)
+		}
+	})
+
+	b.Run("CanonicalAddress", func(b *testing.B) {
+		for i := 0; i < b.N; i++ {
```

Comment:
How about adding `b.ResetTimer()` before this line? If so, it seems to be print each result. But I don't know exactly. Because I cannot compile in my M1 computer. You need to merge the main branch's changes.